aspnetcore/security/authentication/mfa.md

```diff
@@ -49,11 +49,11 @@ To disable support for MFA TOTP, configure authentication using <xref:Microsoft.
 passkeys/FIDO2 is currently:
 
 * The most secure way of achieving MFA.
-* MFA that protects against phishing attacks. (As well as certificiate authentication and Windows for business)
+* MFA that protects against phishing attacks. (As well as certificate authentication and Windows for business)
```

Comment:
This is a preexisting problem, but the "(As well as ...)" sentence doesn't make sense in context. It should probably be made into two extra bullets but then it still wouldn't make sense without something additional.